Tests/Rules/AvoidPositionalParameters.tests.ps1

```diff
@@ -21,9 +21,28 @@ Describe "AvoidPositionalParameters" {
         It "returns no violations" {
             $noViolations.Count | Should -Be 0
         }
-        
+
         It "returns no violations for DSC configuration" {
             $noViolationsDSC.Count | Should -Be 0
         }
     }
+
+    Context "Function defined and called in script, which has 3 or more positional parameters triggers rule." {
+        It "returns avoid positional parameters violation" {
+            $sb=
+            {
+                Function Foo {
+                param(
+                    [Parameter(Mandatory=$true,Position=1)] $A,
+                    [Parameter(Position=2)]$B,
+                    [Parameter(Position=3)]$C)
+                }
+                Foo "a" "b" "c"
+            }
+            $warnings = Invoke-ScriptAnalyzer -ScriptDefinition "$sb"
+            $warnings.Count | Should -BeGreaterThan 0
+            $warnings.RuleName | Should -Contain "PSAvoidUsingPositionalParameters"
```

Comment:
the variable `$violationName = "PSAvoidUsingPositionalParameters"` defined at the top should be re-used

Reply:
Fixed.